vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.4.18",
+  "version": "2.4.19",
```

Comment:
The package-lock file needs to be checked in also.

Reply:
updated